### ISSUE TITLE
The root object, the one that implements the ObjectManager interface, should not appear in the result of a GetManagedObjects call

### ISSUE THREAD

**Issue by mulkieran** [stratis-storage/project#216]
It currently does, so we should try to figure out a way to remove it.

**Comment by mulkieran** [comment]
It looks to me like this would be a one-line change in dbus-rs: https://github.com/diwic/dbus-rs/blob/master/dbus/src/tree/objectpath.rs#L219. What we need is the proof that what's happening now does not meet the specification.

**Comment by mulkieran** [comment]
Here is the spec: https://dbus.freedesktop.org/doc/dbus-specification.html#standard-interfaces-objectmanager

```
All returned object paths are children of the object path implementing this interface, i.e. their object paths start with the ObjectManager's object path plus '/'. 
```

**Comment by mulkieran** [comment]
We can make this a fairly low priority, but, of course, it's not  a lot of code changes either.

**Comment by mulkieran** [comment]
Upstream PR: https://github.com/diwic/dbus-rs/pull/277.

**Comment by mulkieran** [comment]
Merged. I'll keep this open until the next dbus-rs release, but there's nothing more for us to do.

**Comment by mulkieran** [comment]
New version 0.9 was released September 18, 2020.

**Comment by mulkieran** [comment]
We can just wait on the Fedora release: https://src.fedoraproject.org/rpms/rust-dbus/commits/master.

**Comment by mulkieran** [comment]
bz: https://bugzilla.redhat.com/show_bug.cgi?id=1880397

**Comment by mulkieran** [comment]
New version 0.9 is now available.

**Comment by mulkieran** [comment]
The problem is that the "tree" submodule which our D-Bus API relies on extensively has been moved out. We think it will be a big pain to move up.

**Comment by jbaublitz** [comment]
Is this addressed by the multithreading PR?

**Comment by mulkieran** [comment]
By bumping dbus-rs to 0.9 in the multi-threading PR, it picks up the fix.
